test/test_functions.py

```diff
@@ -2,7 +2,7 @@
 import testindicators 

```

Comment:
What is testindicators

Reply:
Testing module, same as indicators but strictly for testing.. it’s built in the unit.sh script before tests are ran